skyportal/tests/api/test_photometry.py

```diff
@@ -740,68 +735,6 @@ def test_token_user_update_photometry_groups(upload_data_token_two_groups,
     assert "Insufficient permissions" in data["message"]
 
 
-def test_post_same_alertid_update_groups(upload_data_token_two_groups,
```

Comment:
I see your logic in wanting to pull this out of here and move to an AlertHandler -- agreed, let's do that. The same handler will then deal with filter id's, duplicate alert_id's etc. 
K would then be like "hey, AlertHandler, alert `candid` for `objectId` passed filter `filter_id` of program `program_id` (or whatever we're calling it, will refresh my memory in the coming days on this for sure :) and here are all new photometry points for that `objectId` since it passed that filter last time". 
Need to think a bit on the details not to duplicate too much data posted in api calls, but let's come back to it a bit later.

Reply:
Awesome glad you agree with this. @profjsb expressed similar views on the standup this AM

Reply:
Note, I would argue that AlertHandler is generic enough and should still be part of SP and only a F-specific extension.

Reply:
Yes, I think `AlertHandler` would be useful for LSST